README.md

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
I can't select it, but a few lines up there's the Xcode version. Should we mark that as Xcode 11.4?